src/services/search/search_result.ts

```diff
@@ -17,7 +21,7 @@ class SearchResult {
         return this.notePathArray[this.notePathArray.length - 1];
     }
 
-    computeScore(fulltextQuery, tokens) {
+    computeScore(fulltextQuery: string, tokens: string[]) {
         this.score = 0;
```

Comment:
this can be removed

Reply:
with this, I was referint to the this.score = 0 expression, as it has been declared in the constructor

Reply:
@Micosilent , I think this could cause issues if `computeScore` is called multiple times, causing the score to keep increasing.

For the scope of these conversions PR I would prefer to avoid any refactoring apart from adding types in order to reduce the risk of regressions.

As we get more accustomed to the code base, we can start doing refactoring as well.